tensorboard/plugins/profile/tf_op_profile/tf-op-table.html

```diff
@@ -254,11 +264,13 @@
       <template
         is="dom-repeat"
         items="{{_getKChildren(node, childrenCount, showP90, level)}}"
+        sort="{{_sort(byWaste)}}"
```

Comment:
I think you mean `[[` since you can't two-way bind

Reply:
Done.